src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,74 +1,68 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
 package frc.robot.subsystems;
 
-import com.revrobotics.CANSparkMax;
-import frc.robot.Config;
-import com.revrobotics.SparkPIDController;
 import com.revrobotics.CANSparkBase.IdleMode;
-import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import static frc.robot.subsystems.IntakeStates.Modes.*;
+import static frc.robot.subsystems.IntakeStates.States.*;
+
+import com.revrobotics.CANSparkMax;
 
 import edu.wpi.first.math.filter.Debouncer;
 import edu.wpi.first.networktables.BooleanPublisher;
 import edu.wpi.first.networktables.NetworkTable;
 import edu.wpi.first.networktables.NetworkTableInstance;
 import edu.wpi.first.networktables.PubSubOption;
+import edu.wpi.first.networktables.StringPublisher;
 import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Config;
+import frc.robot.subsystems.IntakeStates.Modes;
+import frc.robot.subsystems.IntakeStates.States;
 
-import com.revrobotics.CANSparkLowLevel.MotorType;
-
-
-public class IntakeSubsystem extends SubsystemBase {
+/** Add your docs here. */
+public class IntakeSubsystem extends SubsystemBase{
+    private CANSparkMax m_intake;
+    private IntakeStates intakeStates = new IntakeStates();
 
-    private DigitalInput sensor7; //DigitInput instance
-    private DigitalInput sensor8;
-    private DigitalInput sensor9;
+    private DigitalInput sensor7;//Back
+    private DigitalInput sensor8;//Center
+    private DigitalInput sensor9;//Front
```

Comment:
I'm not sure this is the correct order for back, center, front. They got reordered like 5 days ago, I know 8 is not in the center. There are tags put onto the wires close to the sensors that identify which DIO port is used for that sensor. 

There are other places in the code that would also be effected by this.

Reply:
This was the last configuration that worked, in the code there is a specific way to input the values from the sensors, that's how i determined the label of them; besides if they changed can change them once I'm with the robot.

Reply:
btw, that works assuming that the back is the sensor closest to the shooter and the front it the closest to the ground

Reply:
I'm certain the isBackActive variable in IntakeStates is correctly assigned to DIO 7. 

I'm not sure if the variables isForntActive and isCenterActive are correctly assigned to DIO ports matching the front and center switches.